config/initializers/configuration.rb

```diff
@@ -12,7 +12,8 @@
   'slotMinutes' => 15,
   'dragOpacity' => 0.5,
   'selectable'  => true,
-  'timeFormat'  => "h:mm t{ - h:mm t}"
+  'timeFormat'  => "h:mm t{ - h:mm t}",
+  'defaultColor' => '#3B91AD'
```

Comment:
All other variables in this config uses camelCase. Why should it use snake_case?